mod_test.ts

```diff
@@ -1126,3 +1192,55 @@ Deno.test("jsr specifiers - no lockfile", async (t) => {
     assertEquals(ns.join("a", "b"), join("a", "b"));
   });
 });
+
+Deno.test("ignores local css, uses default loader", async (t) => {
+  await testLoader(t, LOADERS, async (esbuild, loader) => {
+    if (esbuild === PLATFORMS.wasm) return;
+    const res = await esbuild.build({
+      ...DEFAULT_OPTS,
+      format: undefined,
+      plugins: [...denoPlugins({ loader })],
+      entryPoints: ["./testdata/basic.css"],
+      bundle: true,
+    });
+    assertEquals(res.warnings, []);
+    assertEquals(res.errors, []);
+    assertEquals(res.errors, []);
+    assertEquals(res.outputFiles.length, 1);
+    const output = res.outputFiles[0];
+    assertEquals(output.path, "<stdout>");
+    assertEquals(
+      output.text,
+      `\
+/* testdata/basic.css */
+.test {
+  color: red;
+}
+.test-svg {
+  background-image: url("data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Ccircle%20r%3D'10'%20cx%3D'10'%20cy%3D'10'%20fill%3D'red'%20%2F%3E%3C%2Fsvg%3E");
+}
+`,
+    );
+  });
+});
+
+Deno.test("ignores local txt, uses default loader", async (t) => {
```

Comment:
This test shows the default loader will be used for local txt files. Notice how it works the same as the old TXT_PLUGIN test without needing that plugin. That's because there is a default loader for txt files in esbuild.